server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -316,6 +314,12 @@ public class Node implements Closeable {
         }
     }, Setting.Property.NodeScope);
 
+    public static final Setting<ByteSizeValue> NODE_SEARCH_CACHE_SIZE_SETTING = Setting.byteSizeSetting(
+        "node.search.cache.size",
```

Comment:
We need to make sure we're consistent with the naming here. Assuming I didn't know what this is, it's immediately obvious to me how this is different from [`"indices.queries.cache.size"`](https://github.com/opensearch-project/OpenSearch/blob/86f8dd65a45810ae671ee1f607a2d5287649d66b/server/src/main/java/org/opensearch/indices/IndicesQueryCache.java#L75). Rabi's PR for the stats API calls this "file_cache". I don't have an immediate answer right now but we need to be clear and consistent across all user facing APIs here.

Reply:
I named it with what logically felt right, since the, but I totally missed this property. I can see how it can get confusing.
We need a way to denote these are search node properties for it to be completely clear.
How about `search_node.xxx` for these properties?

Reply:
`node.search_node` just seems redundant :)

We can probably stick with "node.search.cache.size" since it's trivial to change this while we're still under the feature flag.

Reply:
Ideally the settings should be defined as `node.<role>.xxx`, which is where we are at. I think we still need to somehow clarify `indices queries cache` is unrelated to `search node cache`.  I agree with moving forward under the feature flag, giving a bit more flexibility.